### DISCUSSION TITLE
Change the language for TMDB data

### DISCUSSION THREAD

**Discussion by BapWlo** [meisnate12/Plex-Meta-Manager#605] [Q&A] [answered]
Just set up PMM, had to do a little bit of thinking to understand it, but working in IT it wasn't too hard. Loving it so far !

I was just wondering : can you change the language for the metadata pulled from TMDB ? 
As there's a parameter for TVDB, the actors playlists have a french (I'm french) description, but I can't find a way to make it work for TMDB

Has anyone run into the same issue ? If so, what's the solution ? There's probably a language parameter I didn't see...

**Comment by meisnate12** [comment, marked as answer]
there is lol its not under setting its under the tmdb section in the config 

https://github.com/meisnate12/Plex-Meta-Manager/wiki/TMDb-Attributes

**Comment by BapWlo** [comment]
I'm such a dumba** ! 
Thanks !

**Comment by snickers2k** [comment]
yeah, i didn't found that one neither ... problem is more the not searchable github-wiki, than pmm's location for tmdb_language :(

are there any other good wiki free alternatives maybe, someone?
apiary maybe?

**Reply by meisnate12** [reply]
you can search the whole repo and then select just the wiki results

https://github.com/meisnate12/Plex-Meta-Manager/search?q=tmdb+language&type=wikis
